timemachine/fe/atom_mapping.py

```diff
@@ -347,10 +350,7 @@ def chiral_filter(trial_core):
         passed = not has_chiral_atom_flips(np_core, chiral_set_a, chiral_set_b)
         return passed
 
-    if enforce_chiral:
-        filter_fxn = chiral_filter
-    else:
-        filter_fxn = lambda core: True
+    filter_fxn = chiral_filter if enforce_chiral else lambda _: True
```

Comment:
Would prefer we keep theis as two lines, this is harder for me to read.

Reply:
Will revert, this diff is an unintentional artifact of an earlier revert.